azure-communication-ui/chat/build.gradle

```diff
@@ -72,6 +72,11 @@ dependencies {
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
     implementation "androidx.activity:activity-compose:$androidx_activity_compose_version"
+    implementation "androidx.compose.foundation:foundation:$compose_version"
+    implementation "androidx.compose.material:material:$compose_version"
```

Comment:
Can not bring Material in

Reply:
actually, material is ok, since in fluent_ui transitive.

material3 ones below are not, can't bring in.

Reply:
refacored `AcsChatBackButton` and `ChatCompositeActionBar` to work without Material3